3ds/notifications.go

```diff
@@ -0,0 +1,57 @@
+package friends_3ds
+
+import (
+	database_3ds "github.com/PretendoNetwork/friends-secure/database/3ds"
+	"github.com/PretendoNetwork/friends-secure/globals"
+	nex "github.com/PretendoNetwork/nex-go"
+	nexproto "github.com/PretendoNetwork/nex-protocols-go"
+)
+
+// Notifications that are used in other files with no main file
```

Comment:
Maybe _all_ notifications should be placed in their own files? For example putting `SendUserWentOfflineNotificationsGlobally` into `send_user_went_offline_notification.go` as simple `SendUserWentOffline`. Or at least a `notifications.go` file for each console? That feels cleaner than having some notification functions in a globals file and some in other files, and Go convention is to keep one file per function anyway?

Reply:
Split them into different files.